cinema/serializers.py

```diff
@@ -55,9 +58,11 @@ class Meta:
 
 
 class MovieSessionSerializer(serializers.ModelSerializer):
+    movie_image = serializers.ImageField(source="movie.image.url", read_only=True)
+
     class Meta:
         model = MovieSession
-        fields = ("id", "show_time", "movie", "cinema_hall")
+        fields = ["id", "movie", "movie_image", "session_time", "available_seats"]
```

Comment:
The field `session_time` in the `MovieSessionSerializer` does not match any field in the `MovieSession` model. It should be `show_time` to align with the model definition.